DEVNOTES.md

```diff
@@ -137,13 +139,7 @@ After opening the project in IntelliJ, double check that the Java SDK is properl
 * In the `Project` section, make sure the project language level is set to 11.0 as Ignite makes use of several Java 11
   language features
 
-Ignite uses machine code generation for some of it's modules. To generate necessary production code, build the project using gradle.
-
-Configure Idea code style (for IntelliJ Idea >= 2019):
-* File -> Settings -> Editor -> Code Style -> Scheme -> gear (Show Scheme Actions) -> Import Scheme -> IntelliJ IDEA code style XML
-* Choose: ${igniteProject}/idea/intellij-java-google-style.xml
-* Import schema
-* Reboot IntelliJ Idea
+Ignite uses machine code generation for some of its modules. To generate necessary production code, build the project using gradle.
```

Comment:
Here's a good example why `using` is bad - is it by using or the project that uses gradle?

Apache Ignite 3 uses machine code generation for some of its modules. To generate necessary production code, build the project by using gradle.

Reply:
That's fine, I'll change it, but please note that I didn't write this sentence in the first place, I just replaced "it's" with "its".